### PR TITLE
Add 4 git course on free-couses-id.md

### PR DESCRIPTION
**Pull request by arif-arrijal**
## What does this PR do?
Add resource(s) 

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [ ] Search for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-id.md [hunk 1]
```diff
@@ -111,8 +111,12 @@
 
 ### Git
 
+* [Apa itu GitHub](https://www.youtube.com/playlist?list=PLCZlgfAG0GXCtwnagWsUzZum1CFZYqrB5) - Sekolah Koding (YouTube)
```
Comment (davorpa) [inline courses/free-courses-id.md:114]: Instructor: Hilman Ramadhan @hilmanski

```markdown
* [Apa itu GitHub](https://www.youtube.com/playlist?list=PLCZlgfAG0GXCtwnagWsUzZum1CFZYqrB5) - Hilman Ramadhan, Sekolah Koding (YouTube)
```

File: courses/free-courses-id.md [hunk 2]
```diff
@@ -111,8 +111,12 @@
...
 * [Belajar Git](https://www.youtube.com/playlist?list=PLuGFxya63u24bmP-ILRaiGeMwZh3PGxW4) - Galih Pratama
+* [Belajar GIT (Source Code Management)](https://www.youtube.com/playlist?list=PL8bBYpHH3RI6BlCzFTMQvt7sGSycUj7S-) - UpKoding (YouTube)
```
Comment (davorpa) [inline courses/free-courses-id.md:116]: Instructor: Eka Putra @ekaputra07

```markdown
* [Belajar GIT (Source Code Management)](https://www.youtube.com/playlist?list=PL8bBYpHH3RI6BlCzFTMQvt7sGSycUj7S-) - Eka Putra, UpKoding (YouTube)
```

File: courses/free-courses-id.md [hunk 3]
```diff
@@ -111,8 +111,12 @@
...
+* [Git & Github](https://www.youtube.com/playlist?list=PLFIM0718LjIVknj6sgsSceMqlq242-jNf) - Web Programming UNPAS (YouTube)
```
Comment (davorpa) [inline courses/free-courses-id.md:117]: Instructor: Sandhika Galih @sandhikagalih

```markdown
* [Git & Github](https://www.youtube.com/playlist?list=PLFIM0718LjIVknj6sgsSceMqlq242-jNf) - Sandhika Galih, Web Programming UNPAS (YouTube)
```

**Review by davorpa — CHANGES_REQUESTED** [review]
:information_source: Try to give credits to instructor as well. It's easy picking up from Channel Info or Video Description :wink:

**Comment by arif-arrijal** [comment]
Hi @davorpa, I have been adding credit for the instructor in this course. could you please check it again? :)

**Review by davorpa — APPROVED** [review]
### Ready to merge :fireworks: :tada:

**Pull request merged**